src/sas/sascalc/dataloader/readers/IgorReader.py

```diff
@@ -12,12 +12,11 @@
 #copyright 2008, University of Tennessee
 #############################################################################
 import os
-import numpy
-import math
-#import logging
+
 from sas.sascalc.dataloader.data_info import Data2D
 from sas.sascalc.dataloader.data_info import Detector
 from sas.sascalc.dataloader.manipulations import reader2D_converter
+import numpy as np
 

```

Comment:
I'll create a ticket that gives guidelines for importing numpy, i.e. "the standard way of important and using numpy is to import numpy as np".